internal/provider/resource_gateway_fqdn.go

```diff
@@ -24,49 +24,49 @@ func resourceGatewayFQDNProxy() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Default:     "name",
-				Description: "Gateway workload name (of no actual significance)",
+				Description: "Gateway workload name.",
 			},
 			"solution_type": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Description: "Gateway name (the fqdn will be <name>.<gateway-domain>)",
+				Description: "Solution type for created contract, displayed [here](https://play.dev.grid.tf/#/contractslist).",
 				Default:     "Gateway",
 			},
 			"description": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Default:     "",
-				Description: "Description field",
+				Description: "Description for the gateway fqdn workload.",
 			},
-			"node": {
+			"node_id": {
 				Type:        schema.TypeInt,
 				Required:    true,
-				Description: "The gateway's node id",
+				Description: "The gateway's node id.",
 			},
 			"fqdn": {
 				Type:        schema.TypeString,
 				Required:    true,
-				Description: "The fully quallified domain name of the deployed workload",
+				Description: "The fully quallified domain name of the deployed workload.",
 			},
 			"tls_passthrough": {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     false,
-				Description: "true to pass the tls as is to the backends",
+				Description: "True to pass the tls as is to the backends.",
```

Comment:
description should be `tls passthrough controls the TLS termination, if false, the gateway will terminate the TLS, if True it will only be terminated by the backend service